keps/sig-node/20191127-cri-containerstats-enhancements.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Enhance ContainerStats message in CRI-API
+authors:
+  - "@fuweid"
+owning-sig: sig-node
+participating-sigs:
```

Comment:
Please add `sig-instrumentation` as this API should be part of metrics pipeline

Reply:
Updated.